cchecker_web/static/js/app/index.js

```diff
@@ -128,6 +128,9 @@ _.extend(App.prototype, {
 
       var req = $.ajax({
         url: self.urlRoot + 'upload',
+        beforeSend: function() {
+          $('.drop-status').html('<div class="alert alert-success"><i class="fa fa-spin fa-fw fa-spinner"> </i>' + "Uploading ..."+ '</div>');
```

Comment:
Do you still need the spinner trigger below as well?

Reply:
nvm i see its a different message

Reply:
That other trigger displays when it finishes on the .done so we could potentially get rid of that.